elastic/kibana
# [Fleet] Fix displaying prerelease labels correctly in the installed integration tab

## Description 

Resolve https://github.com/elastic/kibana/issues/151964

Fix displaying the prerelease label based on the installed version of the package and not the latest one available.

We already use the installed version for the card link so it make sense to use it for the prerelease label too.

Also fix a bug when going directly to the installed section, the prerelease settings was not loaded as it was loaded as a side effect of loading all packages.

Also added a check to only fetch settings if the user has the permissions to do it.

## How to test?

Install a package in a non prerelease version, like `endpoint` here and check that the prerelease badge is not displayed, install a package in a prerelease version and check the badge is displayed.
 
<img width="1379" alt="Screenshot 2023-12-13 at 12 03 34 PM" src="https://github.com/elastic/kibana/assets/1336873/73bbbc87-270c-4cc1-95b8-996a84637b43">

I added a unit test that should cover that, and the rest of the integration list should be covered by cypress tests.
